### PR TITLE
Add MultiField

### PR DESCRIPTION
**Pull request by rihr**
## This PR

- [x] Adds MultiField to lab

## Submission criteria

Please review the below criteria before asking for a review.

### Fulfilled the scope of this repo?

- [x] The same functionality of the component can not be achieved with theming, basic styling or composition of existing components
- [x] The component implements an element of the [DHI Design Guidelines](https://www.figma.com/file/pSfX5GNsa6xhKGbi3DWQtn/DHI-Official-Guidelines) or is otherwise generic enough in functionality and close enough to the DHI CVI that it is likely to find reuse in other projects

### Completness

- [x] Story included

### CODE REVIEW

File: packages/react-components-lab/src/components/MultiField/types.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export interface MultiFieldStyleProps {
+  fontSize: number;
+}
+
+export interface MultiFieldProps {
+  value?: string;
+  onChange: (e: string) => void;
+  length?: number; // !Can't be dynamic!
```
Comment (bertearazvan) [inline packages/react-components-lab/src/components/MultiField/types.ts:8]: So it can? :D

**Review by bertearazvan — APPROVED** [review]
Looking really good. Really nice that you added comments explaining what's going on! ⌨️

File: packages/react-components-lab/src/components/MultiField/MultiField.tsx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  KeyboardEvent,
+  useRef,
+  Fragment,
+  useEffect,
+} from 'react';
+import { Box, TextField, Typography } from '@material-ui/core';
+import { MultiFieldProps } from './types';
+import useStyles from './styles';
+
+// Currently supports a-z, A-Z, 0-9 characters. If others are needed, regex needs to be updated
```
Comment (bertearazvan) [inline packages/react-components-lab/src/components/MultiField/MultiField.tsx:13]: I just inserted some other characters like ~#"' etc and it seems to be alright.

**Review by bertearazvan — APPROVED** [review]

File: packages/react-components-lab/src/components/MultiField/MultiField.tsx (hunk 2 above)
Comment (rihr) [reply packages/react-components-lab/src/components/MultiField/MultiField.tsx:13]: You wouldn't be able to paste codes with them

**Pull request merged**